privacy-protections/click-to-load/index.html

```diff
@@ -6,33 +6,35 @@
   <title>Facebook click to load</title>
 </head>
 <body>
-  <!-- Load the facebook SDK in the recommended way -->
-  <script async defer crossorigin="anonymous" src="https://connect.facebook.net/en_US/sdk.js?XFBML=false"></script>
+<!-- Load the facebook SDK in the recommended way -->
+<div id="fb-root"></div>
+<script async defer crossorigin="anonymous" src="https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v14.0" nonce="1SgoaWbh"></script>
 <p><a href="../../">[Home]</a> ↣ <a href="../">[Privacy Protections Tests]</a> ↣ <strong>[Facebook click to load]</strong></p>
 
-<h2> About ClickToLoad Tests</h2>  
-<p>This page embeds the facebook SDK, and several facebook iFrames. It also attempts to create 
-a variety of social elements using the SDK.</P>
+<h2> About ClickToLoad Tests</h2>
```

Comment:
Nit: Could you make the whitespace inside of tags consistent? (Elsewhere too.) Personally I'd remove the extra spaces everywhere, but I don't mind much either way if it's consistent.

Reply:
Yeah I didn't do a full lint because practically no line would have been left untouched, diff is hard enough to read as-is.  I'll do that through this pass.